src/cpp/session/modules/clang/RCompilationDatabase.cpp

```diff
@@ -1178,10 +1178,17 @@ std::vector<std::string> RCompilationDatabase::baseCompilationArgs(bool isCpp) c
    args.push_back("-nostdinc");
 
    // add Rtools arguments
-   auto rtInfo = rToolsInfo();
-   auto rtArgs = rtInfo.clangArgs(isCpp);
-   args.insert(args.end(), rtArgs.begin(), rtArgs.end());
+   //auto rtInfo = rToolsInfo();
+   //auto rtArgs = rtInfo.clangArgs(isCpp);
+   //args.insert(args.end(), rtArgs.begin(), rtArgs.end());
 
+   // add system include headers as reported by compiler
+   std::vector<std::string> includes;
+   discoverSystemIncludePaths(&includes);
+   for (auto&& include : includes) {
+      FilePath includePath = FilePath(include);
```

Comment:
not strictly necessary to convert from string to file path and back to string, but it does clean up the paths and separators a bit